### PR TITLE
Adds support for Windows Subsystem (by @Si13n)

### PR DESCRIPTION
**Pull request by HorlogeSkynet**
## Description
See title

## Reason and / or context
OS handling missing case

## How has this been tested ?
By @Si13n7

## Types of changes :

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Typo / style fix (non-breaking change which improves readability)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: archey [hunk 1]
```diff
@@ -479,6 +500,11 @@ class Output:
             exit()
 
         for distribution in Distributions:
+            if re.fullmatch(lsbOutput, "openSUSE") or re.fullmatch(lsbOutput, "Ubuntu"):
```
Comment (HorlogeSkynet) [inline archey]: Hey Roy, I'm ready ! @Si13n7

Could you paste here the different outputs of `lsb_release -i -s` and `uname -r` ?
I'd like to optimize this very loop without this test case before merging this PR !

Thx again,
👋

Comment (Si13n7) [reply archey]: Hey Samuel,

here:

| System   | lsb_release -i -s | uname -r           |
| --------- | ----------------- | ----------------- |
| Ubuntu 16.04.3 LTS | Ubuntu | 4.4.0-109-generic  |
| Windows 10 + Ubuntu Subsystem | Ubuntu | 4.4.0-43-Microsoft |

Comment (HorlogeSkynet) [reply archey]: Thanks ! I guess for `openSUSE` case it'd be the same as the second row of your table by replacing `Ubuntu` ?

Comment (Si13n7) [reply archey]: I don't have an openSUSE installation right now, but I still remember that when I tested it the outputs were similar 😄

Comment (HorlogeSkynet) [reply archey]: Well okay 😂
I'll start working on this this evening. 'keep you posted 😉

Comment (Si13n7) [reply archey]: I added the openSUSE subsystem to the list 😉

Comment (HorlogeSkynet) [reply archey]: Ahhh thanks, so fast 😜

Comment (HorlogeSkynet) [reply archey]: Well... I can't push to your fork this time... Do you have a checkbox somewhere around here to allow me to contribute to the fork ? 😕

Comment (Si13n7) [reply archey]: I can't see anything like that...

Comment (HorlogeSkynet) [reply archey]: Wtf 😂 Could you send me a new invitation for this time ? 🙂

Comment (Si13n7) [reply archey]: EDIT: [link removed] (check your emails 😜)

Comment (HorlogeSkynet) [reply archey]: Thanks it's pushed 😜
Tell me what you think about this, I think it's better to check for `Microsoft` anyway...

Comment (Si13n7) [reply archey]: lets measure and compare...

| old | new |
| --- | --- |
| 0.12305116653442383s | 0.11970376968383789s |

Yeah, your way is better! 😜

**Comment by HorlogeSkynet** [comment]
Perfect 😂
Are we OK for merging this then ? 🙂

**Comment by Si13n7** [comment]
Yes of course! 😉

**Pull request merged**